packages/web/config/chain-infos.ts

```diff
@@ -136,6 +136,46 @@ const chainInfos = (
           isStakeCurrency: true,
           isFeeCurrency: true,
         },
+        {
+          type: "cw20",
+          contractAddress: "secret12rcvz0umvk875kd6a803txhtlu7y0pnd73kcej",
+          coinDenom: "ALTER",
+          coinMinimalDenom:
+            "cw20:secret12rcvz0umvk875kd6a803txhtlu7y0pnd73kcej:ALTER",
+          coinDecimals: 6,
+          coinGeckoId: "alter",
+          coinImageUrl: "/tokens/alter.jpg",
+        },
+        {
+          type: "cw20",
+          contractAddress: "secret1yxcexylwyxlq58umhgsjgstgcg2a0ytfy4d9lt",
+          coinDenom: "BUTT",
+          coinMinimalDenom:
+            "cw20:secret1yxcexylwyxlq58umhgsjgstgcg2a0ytfy4d9lt:BUTT",
+          coinDecimals: 6,
+          coinGeckoId: "button",
+          coinImageUrl: "/tokens/butt.png",
+        },
+        {
+          type: "cw20",
+          contractAddress: "secret1qfql357amn448duf5gvp9gr48sxx9tsnhupu3d",
+          coinDenom: "SHD",
+          coinMinimalDenom:
+            "cw20:secret1qfql357amn448duf5gvp9gr48sxx9tsnhupu3d:SHD",
+          coinDecimals: 8,
+          coinGeckoId: "shade-protocol",
+          coinImageUrl: "/tokens/shd.jpg",
```

Comment:
png or svg is preferred...

Reply:
Yes, I'll update soon

Reply:
The SHD png/svg was added to the registry this morning if you need it.
https://github.com/cosmos/chain-registry/tree/master/secretnetwork/images

Reply:
Sienna waiting to be added to the registry in this PR: https://github.com/cosmos/chain-registry/pull/985

Reply:
@JeremyParish69 https://github.com/cosmos/chain-registry/pull/986